federator/src/contracts/IBridgeNft.ts

```diff
@@ -0,0 +1,27 @@
+export class IBridgeNft {
+  nftBridgeContract: any;
+
+  constructor(nftBridgeContract: any) {
+    this.nftBridgeContract = nftBridgeContract;
+  }
+
+  getFederation(): any {
+    return this.nftBridgeContract.methods.getFederation();
```

Comment:
Add .call() to invoke the method
```suggestion
    return this.nftBridgeContract.methods.getFederation();
```

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/257/commits/960a77254356ad684efe45071b291b518e0266f8